src/style.css

```diff
@@ -1 +1,79 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+header {
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: flex;
+  justify-content: flex-end;
+  width: 100%;
+  height: 60px;
+  background: #fff;
+  margin: 0;
+  padding: 0;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+ul {
+  height: 60px;
+  display: flex;
+  justify-content: flex-end;
+  align-items: flex-end;
+  font-family: Roboto, sans-serif;
+  font-style: bold;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: black;
+  padding-right: 40px;
+  margin: 0;
+}
+
+img {
+  position: absolute;
+  top: 10px;
+  left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+nav {
+  display: flex;
+  padding: 0 10px ;
+  height: 60px;
+  justify-content: center;
+  align-items: center;
+}
+
+nav :hover {
+  height: 60px;
+  display: flex;
+  color: #00acdc;
+  justify-content: center;
+  align-items: center;
+}
+
+nav :active {
+  text-decoration: underline 4px solid #00acdc;
+  text-underline-offset: 20px;
+  height: 60px;
+  display: flex;
+  color: #00acdc;
+  justify-content: center;
+  align-items: center;
+}
+
+.is-active {
+  text-decoration: underline 4px solid #00acdc;
```

Comment:
use :after for create this "underline", because now it's not according the design